### PR TITLE
changed story parser to use single lines instead of matching multiple

### PR DESCRIPTION
**Pull request by edewit**
fixes: #301

### CODE REVIEW

File: storeys/src/test/java/ch/vorburger/minecraft/storeys/model/parser/StoryParserTest.java [hunk 1]
```diff
@@ -113,14 +110,14 @@ public static StoryParser getStoryParser() {
...
-        assertEquals(4, storyActionsList.size());
+        assertEquals(9, storyActionsList.size());
         assertEquals(TitleAction.class, storyActionsList.get(0).getClass());
-        assertEquals(MessageAction.class, storyActionsList.get(1).getClass());
-        NarrateAction narrateAction = (NarrateAction) storyActionsList.get(2);
+        assertEquals(MessageAction.class, storyActionsList.get(2).getClass());
+        NarrateAction narrateAction = (NarrateAction) storyActionsList.get(4);
         assertEquals("Piggy", narrateAction.getEntityName());
-        assertEquals(Text.of("Hi there! I'm Piggy.\nWelcome to the storeys mod.  I'll be giving you a quick guided tour now..."),
+        assertEquals(Text.of("Welcome to the storeys mod.  I'll be giving you a quick guided tour now...").concat(Text.NEW_LINE).concat(Text.of("Hi there! I'm Piggy.")),
```
Comment (vorburger) [inline storeys/src/test/java/ch/vorburger/minecraft/storeys/model/parser/StoryParserTest.java:118]: Doesn't this mean that we're parsing it in the wrong order now? Unless I misunderstand, this test seems to assert that the narration text is in the wrong order, based on `parse-test.story`.

Comment (edewit) [reply storeys/src/test/java/ch/vorburger/minecraft/storeys/model/parser/StoryParserTest.java:118]: yeah I don't know why I changed that, I had it right the first time then later I changed, because for some reason I thought it was wrong the first time... I don't know what I was thinking... changed it back

File: web/src/main/java/ch/vorburger/minecraft/storeys/web/StaticWebServerVerticle.java [hunk 2]
```diff
@@ -93,7 +98,7 @@
             } catch (IOException e) {
                 throw new RuntimeException("Couldn't create / update scratch project", e);
             }
-            context.response().send(userId);
+            context.response().end(new JsonObject().put("id", userId).encodePrettily());
```
Comment (vorburger) [inline web/src/main/java/ch/vorburger/minecraft/storeys/web/StaticWebServerVerticle.java:101]: @edewit was this intentionally changed here? Seems unrelated to this.

**Review by vorburger — APPROVED** [review]
LGTM as a next step. We should probably fix #333 next though.

File: web/src/main/java/ch/vorburger/minecraft/storeys/web/StaticWebServerVerticle.java (hunk 2 above)
Comment (edewit) [reply web/src/main/java/ch/vorburger/minecraft/storeys/web/StaticWebServerVerticle.java:101]: oops... yes that was something that was still in my working folder, this fixes a javascript error in scratch
